packages/graphql/src/schema-model/entity/model-adapters/ConcreteEntityAdapter.ts

```diff
@@ -128,6 +187,10 @@ export class ConcreteEntityAdapter {
         return this._plural;
     }
 
+    public get upperFirstPlural(): string {
```

Comment:
Maybe this could be just a Schema Generator logic, in the moment you know you have to access `upperFirstPlural` it will be easy to just have `upperFirst(concreteEntityAdapter.plural)` and avoid having future variations of these combinations `upperFirstSingular`, `lowerFirstPlural` etc...!